server/chat-plugins/chat-monitor.ts

```diff
@@ -672,4 +733,8 @@ export const commands: ChatCommands = {
 process.nextTick(() => {
 	Chat.multiLinePattern.register('/filter (add|remove) ');
 	loadFilters();
+	for (const list of Object.keys(Chat.monitors).filter(x => Chat.monitors[x].monitor)) {
+		(Chat.commands['filter'] as ChatCommands)[`test${list.replace('filter', '')}`] = 'test';
+		if (list.includes('filter')) (Chat.commands['filter'] as ChatCommands)[`test${list}`] = 'test';
+	}
```

Comment:
You could have the syntax be `/filter test [filtername]` to get around the plugin loading order issue.

Reply:
did @Zarel